.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2.1
+orbs:
+  heroku: circleci/heroku@0.0.10
+jobs:
+  run_tests:
+    docker:
+      - image: 'circleci/node:10.0'
+    steps:
+      - checkout
+      - run:
+          name: Run Tests
+          command: echo 'This will eventually run tests'
+  build_and_deploy:
+    docker:
+      - image: 'circleci/node:10.0'
+    steps:
+      - checkout
+      - heroku/install
+      - setup_remote_docker:
+          version: 19.03.8
+      - run:
+          name: Build Docker image and push to Heroku registry
+          command: >
+            docker login --username=$HEROKU_USERNAME --password=$HEROKU_PASSWORD registry.heroku.com
```

Comment:
Where do these environment variables get set?

Reply:
Those are set [here](https://app.circleci.com/settings/project/github/mobilemelody/board-at-home/environment-variables?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fmobilemelody%2Fboard-at-home)

You can get to this place by selecting the project in CircleCI, clicking "Project Settings" on the top-right corner of the page, and then clicking on "Environment Variables" on the left sidebar.